.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @salemove/tm-messenger
```

Comment:
Messenger doesn't seem to own any services that use it https://github.com/search?q=org%3Asalemove+helpstation&type=code&p=1

So either @salemove/tm-quantum , @salemove/tm-engage or @salemove/tm-cerebro (because of statistics service that I heard will be rewritten soon).

Reply:
I've approved this PR. I'm still blocked in merging. Can you approve and merge yourself, I'll be out.